### PR TITLE
Update README, added documents for setup + contributing

### PR DESCRIPTION
**Pull request by mcpenguin**
## Summary of Changes
Updated README and added documents on setting up the development environment and how to contribute to the bot.

## Motivation and Explanation
This makes it easier for new developers to set up their development environment and contribute to the bot.

## Related Issues
#311

### CODE REVIEW

**Review by Picowchew — CHANGES_REQUESTED** [review]
I just left some small comments.

File: docs/SETUP.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+5. In your cloned project, create a `dev` folder in `config`, and create a `vars.json` file in it. Fill the JSON file with the corresponding values for each of the variables as described in the main README.md.
+
+6. Make an `.env` file in the root folder of the project, and put your Discord bot's token. DO NOT REVEAL THIS: ANYONE WITH THIS HAS ACCESS TO YOUR BOT.
+
+7. Run `yarn` in the root folder of your project to install all the dependencies required.
+
+### Running the Bot using Docker
+
+8. Run `yarn image:build` to create and build the Docker image for the bot.
```
[the remark this answers is not in the document]
Comment (mcpenguin) [reply docs/SETUP.md:28]: Are you sure?

File: docs/SETUP.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+> Codey is up! App version: \<app version>
+
+13. To stop the bot, run `yarn stop`. To stop the bot and remove the container, run `yarn clean`.
+
+### Running the bot locally
+
+You can choose to run the bot locally instead of using Docker. This is helpful if Docker is not working as intended.
+
+8. Install the dependencies `ghostscript` and `graphicsmagick` locally. This is required for the resume pdf->image converter functionality of Codey.
```
[the remark this answers is not in the document]
Comment (mcpenguin) [reply docs/SETUP.md:51]: `yarn` is 7 - shouldn't this be `8`?

File: docs/SETUP.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Picowchew) [reply docs/SETUP.md:28]: It is a new section for Docker, so shouldn't it be 1?

File: docs/SETUP.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Picowchew) [reply docs/SETUP.md:55]: Oh I see what you mean. Hmmm, I guess it just seems kind of odd how 13 becomes 8? It might be better as 1?

**Review by Picowchew — APPROVED** [review]
LGTM!

**Pull request merged**